Makefile.toml

```diff
@@ -24,7 +24,7 @@ TWOLITER_ALLOW_BINARY_INSTALL="true"
 
 # Allow Twoliter to be installed by building from sourcecode.
 TWOLITER_ALLOW_SOURCE_INSTALL="true"
-
+TARGET_CC="x86_64-unknown-linux-gnu"
```

Comment:
What is this doing?

Reply:
I didn't mean to add this, should be removed with the latest commit